python/ray/dag/compiled_dag_node.py

```diff
@@ -1564,6 +1583,20 @@ def run(self):
         monitor.start()
         return monitor
 
+    def raise_if_too_many_inflight_requests(self):
+        num_in_flight_requests = (
+            self._execution_index - self._max_finished_execution_index
+        )
+        if num_in_flight_requests > self._max_in_flight_requests:
+            raise ray.exceptions.RayAdagAtMaxCapacity(
+                f"There are {num_in_flight_requests} in-flight requests which "
+                "is more than the max buffer size of the dag: "
```

Comment:
```suggestion
                "is more than the specified  _max_in_flight_requests of the dag: "
```

There is also `max_buffered_results`. User may confuse with it when talking about "max buffer size"